### PR TITLE
Add Node Definition to Glossary

### PR DESCRIPTION
**Pull request by DavidMansolino**
https://www.cyberbotics.com/doc/reference/glossary?version=add-node-definition-to-glossary

### CODE REVIEW

**Comment by DavidMansolino** [comment]
:+1: 2    (feel free to edit the definition)

File: reference/glossary.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ Exceptionally, some *actuators* may also be *sensors* if they make sensory measu
 
 **Ancestor**: When referring to a *node*, the *ancestor* of a *descendant* is a *node* which hierarchically contains the *descendant* in the *scene tree*, at any relative depth.
 
+**Base node**: A *base node* is a node a Webots built-in node.
```
Comment (DavidMansolino) [inline reference/glossary.md]: This sentence looks wrong.

Comment (DavidMansolino) [reply reference/glossary.md]: Probably `a node` should be removed, right?

Comment (omichel) [reply reference/glossary.md]: Yes...

**Comment by omichel** [comment]
One more review is needed here.

**Pull request merged**